app/components/AddThis.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+
+interface Props {
+  title: string;
+  path: string;
+}
+
+export default class Header extends React.PureComponent<Props> {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    window.addthis.share();
+  }
+
+  render() {
+    const url = `https://www.aliemcards.com${this.props.path}`;
+    return (
+      <div className="addthis_share">
+        <div className="addthis_button">
+          <i className="addthis_icon_facebook" />
+          <div
```

Comment:
Why is this div here? Can we put these data attributes on the enclosing div?

Reply:
The div with "addthis_share_button" has to be separate from the icon or the addthis script adds a link to the icon image without the appropriate data-attributes, even if the data-attributes are on the containing element.